KnightForge/theme.json5

```
@@ -0,0 +1,24 @@
+/* SPDX-FileCopyrightText: yar2000T <https://github.com/yar2000T> */
+/* SPDX-FileCopyrightText: Simon Schneegans <code@simonschneegans.de> */
+/* SPDX-License-Identifier: CC-BY-4.0 */
```

Do you really want this to be CC-BY for this file? I think CC0 would be more suitable for such kind of a simple config file. The original one from the Minecraft theme is CC0 as well, so you can also omit my name to keep this cleaner.

```suggestion
// SPDX-FileCopyrightText: yar2000T <https://github.com/yar2000T>
// SPDX-License-Identifier: CC0-1.0
```

---

I changed it, sorry for the inconvenience. I just never work with such type of license